lib/active_admin_import/model.rb

```diff
@@ -1,5 +1,7 @@
 # encoding: utf-8
 
+require 'rchardet'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.